src/arkit.cs

```diff
@@ -1986,13 +1991,21 @@ interface ARSkeleton3D {
 		[Protected, Export ("jointLocalTransforms")]
 		IntPtr RawJointLocalTransforms { get; }
 
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
 		[Export ("modelTransformForJointName:")]
 		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
-		Matrix4 GetModelTransform ([BindAs (typeof (ARSkeletonJointName))] NSString jointName);
+		Matrix4 GetModelTransform (NSString jointName);
+
+		[Wrap ("GetModelTransform (jointName.GetConstant()!)")]
+		Matrix4 GetModelTransform (ARSkeletonJointName jointName);
```

Comment:
what was the reason to remove `[BindAs]` ?

Reply:
same